src/learn/qml/MuseScore/Learn/LearnPage.qml

```diff
@@ -167,6 +168,7 @@ FocusScope {
             navigation.name: "Get started"
             navigation.panel: navTabPanel
             navigation.column: 1
+            onClicked: searchField.visible = true
```

Comment:
Code like this does not really fit into QML's declarative style. The stylistically "correct" way is to bind the visibility of the search field to the `currentIndex` of the `tabBar`. So, _inside the `SearchField { … }`_, you add:
```qml
visible: tabBar.currentIndex !== 3 // Not visible for Classes tab
```